mapping.json

```diff
@@ -0,0 +1,111 @@
+{
+    "schema_id": "cof_r2_assessment",
```

Comment:
should we call it `cof_r2w2_assessment` as it maps directly then to the short names of fund/round

Reply:
sounds sensible :)